vendor/bat-native-ledger/src/bat/ledger/internal/bat_helper.cc

```diff
@@ -1917,6 +1934,19 @@ bool getJSONAddresses(const std::string& json,
   return !error;
 }
 
+bool getJSONMessage(const std::string& json,
```

Comment:
please use base JSONReader for this one. We want to avoid rapidjson

Reply:
It has the same implementation as other functions in this class (like `getJSONResponse`, `getJSONGrant` etc). If we want to avoid rapidjson we should make changes everywhere in this class. But I would do it out of scope of this PR.

Reply:
Plus to what was said above `JSONReader` is a chromium class which was initially avoided from usage to make the library SDK look like and don't use chromium dependencies there.

Reply:
So @NejcZdovc we are going to create separate issue for this. Does it sound good for you?

Reply:
sounds good